demisto_sdk/commands/content_graph/commands/update.py

```diff
@@ -68,58 +72,75 @@ def update_content_graph(
         marketplace (MarketplaceVersions): The marketplace to update.
         use_git (bool): Whether to use git to get the packs to update.
         imported_path (Path): The path to the imported graph.
-        use_local_import (bool): Whether to use the current import folder to import graph.
         packs_to_update (List[str]): The packs to update.
         dependencies (bool): Whether to create the dependencies.
         output_path (Path): The path to export the graph zip to.
     """
-    if not use_local_import and not imported_path and not use_git:
+    if not imported_path and not use_git:
         logger.info("No arguments were given, using git")
         use_git = True
     git_util = GitUtil()
+    is_external_repo = is_external_repository()
 
-    if is_external_repository():
+    if is_external_repo:
         packs_to_update = get_all_repo_pack_ids()
     packs_to_update = list(packs_to_update) if packs_to_update else []
     builder = ContentGraphBuilder(content_graph_interface)
-    if not should_update_graph(
-        content_graph_interface, git_util, imported_path, use_local_import
+    is_graph_alive = neo4j_service.is_alive()
+    if is_graph_alive and not should_update_graph(
+        content_graph_interface, use_git, git_util, imported_path, packs_to_update
     ):
         logger.info(
-            f"Content graph is up to date, no need to update. UI representation is available at {NEO4J_DATABASE_HTTP} "
+            f"Content graph is up to date, no need to update. Make sure to add/commit your changes. UI representation is available at {NEO4J_DATABASE_HTTP} "
             f"(username: {NEO4J_USERNAME}, password: {NEO4J_PASSWORD})"
         )
         return
     builder.init_database()
+    use_local_import = True
+    try:
+        if imported_path:
+            logger.info(f"Importing graph from {imported_path}")
+            content_graph_interface.clean_import_dir()
+        else:
+            logger.info(
+                f"Importing graph from local folder: {str(content_graph_interface.import_path)}"
+            )
+
+        is_graph_up_to_date = content_graph_interface.import_graph(imported_path)
+        if not any([imported_path, is_graph_up_to_date]):
+            use_local_import = False
+    except Exception:
```

Comment:
It can raise an exception if the query fails.

In line 132, it will go to @recover_if_fails decorator. I think we can remove it and let recover_if_fails to handle this as well